server/lomas_server/dp_queries/dp_libraries/smartnoise_synth.py

```diff
@@ -18,21 +18,29 @@
 
 from lomas_server.admin_database.admin_database import AdminDatabase
 from lomas_server.constants import (
-    DEFAULT_DATE_FORMAT,
```

Comment:
should also be removed from constants.py if removed here

Reply:
Yes, I propose to do this with issue 268 (add constants and remove unused).

Reply:
I think it is cleaner and easier to do it when we stop using them in the codebase.

Reply:
ok will remove